tests/testDipoleFitter.py

```diff
@@ -74,34 +83,35 @@ def __init__(self, xc=np.array([65.3, 24.2]), yc=np.array([38.6, 78.5])):
             xcenNeg=self.xc - self.offsets,
             ycenNeg=self.yc - self.offsets,
             flux=self.flux, fluxNeg=self.flux,
+            noise=2.,  # Note the input noise - this affects the relative tolerances used.
             gradientParams=self.gradientParams)
 
         self.catalog = self.testImage.detectDipoleSources()
+        self.rtol = 0.01  # This is okay given the default noise of 2. set above.
```

Comment:
I'd include a bit more from your reply to my earlier comment. Something like this (paraphrasing from your comment).

```
# Because of the noise added to the images, 1% tolerance is a useful test of algorithm robustness.
# The centroids are much better constrained than the fluxes, and 1% often is not achieved for the flux measurements.
# This is here to check against regression - we don't want future changes to make things worse.
```